glam/api/migrations/0023_drop_alter_MV.py

```diff
@@ -0,0 +1,99 @@
+# Generated by Django 3.1.13 on 2022-03-04 01:21
```

Comment:
This migration file is dropping the old MV and renaming the backup to make it the effective one. Is this file supposed to be in this PR?

Reply:
Without this file, the tests are failing as discussed in https://mozilla.slack.com/archives/C01QMME608M/p1646873336029739

Reply:
Got it. Thanks!
I hadn't read that and got confused by your first comment on this PR.